src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--month-length--29">
+        <div class=" calendar__day calendar__day--first-day--sun"></div>
```

Comment:
move `calendar__day--first-day--` modifier to the whole calendar block (logic remain the same - left margin for the first day)

Reply:
cause you modify the whole calendar appearance, and it would be grate to have all modifiers on calendar block